src/tests/test.js

```diff
@@ -0,0 +1,172 @@
+const http = require('http')
+const path = require('path')
+const url = require('url')
+const puppeteer = require('puppeteer')
+const serveHandler = require('serve-handler')
+const testKeys = require('recaptcha-test-keys')
+const Grecaptcha = require('grecaptcha')
+
+// Encapsulate the asynchronous starting and stopping of a simple HTTP server at http://localhost:3000. This is needed
+// because reCAPTCHAs don't work from within pages loaded directly from the file system.
+const server = {
```

Comment:
Could you move these objects(`api`, `browser` and `server`) to a test-helper file? Leaving on the test file only the assertions?